css/src/elementor.css

```diff
@@ -0,0 +1,73 @@
+/*
+ * Style adaptions for the Elementor integration
+ * Mostly fills in the gaps for pieces that normally receive their styles through the block-editor
+*/
+
+:root {
+	--yoast-elementor-color-paragraph: #555D66;
+}
+
+.yoast {
+	font-family: var(--yoast-font-family) !important;
```

Comment:
Any way of handling this with increased specificity (like `&&&`) rather than `!important`?

Reply:
Deliberately chose to use an !important in this case, as we need to overrule Elementor's font in all our stuff to mimic the Gutenberg look. But a slightly more polite approach seems to tackle everything either, so I removed it!